homeworks/szabolcsgesztesi/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,18 @@
+
+name = input("Kérlek add meg a neved: ")
+name = name.strip().capitalize()
```

Comment:
nem hiba, de ha akarod ez mehet 1 sorra is:

name = input("Kérlek add meg a neved: ").strip().capitalize()

Reply:
Javítva!